modules/optflow/src/.clang-format

```diff
@@ -0,0 +1,65 @@
+---
+Language:        Cpp
```

Comment:
Suggest removing this file.  It seems like cruft, I don't see a large precedence for clang-format files in opencv.